modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -203,6 +204,9 @@ export interface AddressInfo {
 
 export interface UnspentInfo {
   address: string;
+  amount: number;
+  txid: string;
+  n: number;
```

Comment:
The preferred term to refer to the position of an output on a transaction is `vout`. I would suggest that we use that instead of `n` unless we are committed already.